### PR TITLE
NE Scraper

### PR DESCRIPTION
**Pull request by aed3**

### CODE REVIEW

File: src/events/crawler/scrapers/USA/NE/index.js [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+// const UNASSIGNED = '(unassigned)';
+
+const scraper = {
+  state: 'NE',
+  country: 'USA',
+  url: 'https://www.lincoln.ne.gov/city/covid19/',
+  type: 'list',
+  aggregate: 'county',
+  maintainers: [maintainers.aed3],
+
```
Comment (lazd) [inline src/events/crawler/scrapers/USA/NE/index.js]:
```suggestion
source: {
  name: 'City of Lincoln, Nebraska COVID-19 Response'
},
```

File: src/events/crawler/scrapers/USA/NE/index.js [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    let counties = [];
+    const $ = await fetch.page(this.url);
+
+    const listItems = $('.stats-section ul li:contains("Confirmed cases in Nebraska") li');
+
+    listItems.each((index, li) => {
+      const text = $(li).text();
+
+      counties.push({
+        county: text.slice(0, text.search(/[^\w\s]/)),
```
Comment (lazd) [inline src/events/crawler/scrapers/USA/NE/index.js]: Does this handle the `Douglas County/Omaha` row?

Comment (aed3) [reply src/events/crawler/scrapers/USA/NE/index.js]: Yes

File: src/events/crawler/scrapers/USA/NE/index.js [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+// Set county to this if you only have state data, but this isn't the entire state
+// const UNASSIGNED = '(unassigned)';
+
+const scraper = {
+  state: 'NE',
+  country: 'USA',
+  url: 'https://www.lincoln.ne.gov/city/covid19/',
+  type: 'list',
+  aggregate: 'county',
+  maintainers: [maintainers.aed3],
```
Comment (lazd) [inline src/events/crawler/scrapers/USA/NE/index.js]:
```suggestion
  maintainers: [maintainers.aed3],
  priority: 1,
```

**Pull request merged**